lib/translate/translate

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env sh
 set -e
 
-optLevel=${2:-Oz}
-
 llvm-extract-12 --delete --ralias="$1" --rfunc="$1" symbolica.bc |
-opt-12 --strip-debug --loweratomic --${optLevel} --scalarizer -o=.symbolica.bc
+opt-12 --strip-debug --loweratomic "$2" --scalarizer -o=.symbolica.bc
```

Comment:
I think we should merge this change in to master first as it will have some implications on the hosted/playground side.